integration-cli/docker_cli_run_test.go

```diff
@@ -2538,3 +2538,7 @@ func (s *DockerSuite) TestRunNetworkFilesBindMount(c *check.C) {
 		c.Fatalf("expected resolv.conf be: %q, but was: %q", expected, actual)
 	}
 }
+
+func (s *DockerSuite) TestRunCapAddSYSTIME(c *check.C) {
+	dockerCmd(c, "run", "--cap-drop=ALL", "--cap-add=SYS_TIME", "busybox", "sh", "-c", "date --set `date '+%H:%M:%S'` > /dev/null", "true")
```

Comment:
@sunyuan3 you don't need `true`, but you do need to check for the `err` returned by `dockerCmd`, and if there was an error, you should print out an error message with Fatalf, including the output of dockerCmd.
```
out, err := dockerCmd(...)
if err != nil {
    c.Fatalf("could not set time: %s", out)
}
```